src/TwigExtension.php

```diff
@@ -211,5 +213,23 @@ public function markdownToHtml($body): string {
     return $Parsedown->text($body);
   }
 
+  /**
+   * Converts Markdown to HTML.
+   *
+   * @param string $edtfString
+   * @param string $lang
+   *
+   * @return string
+   */
+  public function edtfToHumanDate(string $edtfString, string $lang = 'en'): string {
+    $parser = \EDTF\EdtfFactory::newParser();
```

Comment:
hello! Replace `\EDTF\EdtfFactory` with an `use` statement so you do not need the full class path! (also good night)